app/points/calculate-points-for-tier.js

```diff
@@ -1,9 +1,10 @@
 const TierCounts = require('../../app/points/domain/tier-counts')
 const assertObjectType = require('../../app/points/domain/validation/assert-object-type')
 const assertNumber = require('../../app/points/domain/validation/assert-number')
+const calculateWeighting = require('./calculate-weighting')
 
-var calculateWeightedPoints = function (count, points, weighting = 1) {
-  return count * (points * weighting)
+var calculateWeightedPoints = function (count, points, weighting = 0) {
+  return count * (points * calculateWeighting(weighting))
```

Comment:
Had a look at the `calculateWeighting` function. Seems odd that a weighting percentage of 0 multiplies the points value by one, but a weighting of 100 multiples the value by 0? Also, the third calculation of weighting seems backwards?